judgels-backends/uriel/uriel-app/src/main/java/judgels/uriel/contest/clarification/ContestClarificationStore.java

```diff
@@ -33,9 +33,20 @@ public ContestClarification createClarification(String contestJid, ContestClarif
     public Optional<ContestClarification> updateClarificationAnswer(
             String contestJid,
             String clarificationJid,
-            String answer) {
+            String answer, boolean allowDuplicates) {
```

Comment:
Duplicates are never allowed

Reply:
OK, `allowDuplicates` param will be removed.

I think `updateClarificationAnswer` is quite confusing, because I'd assume that it is for editing a clarification answer, when in reality it is used for setting an answer to a clarification. Should I rename the method instead to `answerClarification`?

Reply:
Yes.